.github/workflows/_diffcalc_processor.yml

```diff
@@ -0,0 +1,250 @@
+name: "🔒diffcalc (do not use)"
```

Comment:
I've done this because gha doesn't have any method to reorder/organise workflows, and this at least pushes it to the bottom of the list: https://github.com/smoogipoo/osu/actions/workflows/diffcalc.yml